### PR TITLE
Global system select

### PR DESCRIPTION
**Pull request by Kasmadei**
@blcham 
Added dropdown for system select.
After select, we reorder systems list where on first place we put selected system.
Mostly done, need to fix BE response to finish correct filter for fta trees.
Screenshot 2024-05-09 at 8 27 53 PM

### CODE REVIEW

**Review by blcham — APPROVED** [review]
LGTM

**Comment by blcham** [comment]
@LaChope please have a peek :)

File: src/components/appBar/AppBar.tsx [hunk 1]
```diff
@@ -27,11 +31,29 @@ const AppBar = ({ title, showBackButton = false, topPanelHeight }: Props) => {
...
+  const [systems, setSystems] = useState<System[]>([]);
+
   const isMenuOpen = Boolean(anchorEl);
 
+  React.useEffect(() => {
+    const fetchSystems = async () => {
+      systemService
+        .findAll()
+        .then((value) => setSystems(value))
+        .catch((reason) => showSnackbar(reason, SnackbarType.ERROR));
```
Comment (LaChope) [inline src/components/appBar/AppBar.tsx:49]: I would not show such a raw error to the end user, maybe something like "Unexpected error occurred".

**Review by LaChope — COMMENTED** [review]
Just see minor comment.

File: src/components/appBar/AppBar.tsx [hunk 2]
```diff
@@ -27,11 +31,29 @@ const AppBar = ({ title, showBackButton = false, topPanelHeight }: Props) => {
   const { classes } = useStyles();
   const history = useNavigate();
   const { i18n } = useTranslation();
+  const { t } = useTranslation();
```
[the remark this answers is not in the document]
Comment (Kasmadei) [reply src/components/appBar/AppBar.tsx:34]: Sure, i'll fix it in next PR

File: src/components/appBar/AppBar.tsx (hunk 1 above)
Comment (Kasmadei) [reply src/components/appBar/AppBar.tsx:49]: Thanks, i'll change it

**Pull request merged**